src/imgui/colors.h

```diff
@@ -0,0 +1,11 @@
+#ifndef MYPROJECT_COLORS_H
+#define MYPROJECT_COLORS_H
+
+#include <imgui.h>
+
+namespace colors {
+const ImVec4 Red{ .7F, .0F, .0F, 1.0F };
+const ImVec4 Green{ .0F, .8F, .0F, 1.0F };
```

Comment:
this naming comes from the rayTracer?

Reply:
What naming? colors, Red and Green are mine. ImVec4 is the imgui 4d vec. It's using rgba values. We can create a more robust Color class like we did on the ray tracer. Then this type can be internally converted to ImVec4 when it's time to print. This way the client code never sees a imgui type. We can also add more colors XD

Reply:
they could really use a better naming

Reply:
I meant the ImVec4. :D